### DISCUSSION TITLE
运行 main.py 的时候报错了，说完缺少这个模块。 求问大佬接下来怎么做

### DISCUSSION THREAD

**Discussion by Sqyht1** [binary-husky/gpt_academic#594] [Q&A]
Traceback (most recent call last):
  File "/Users/apple/main.py", line 209, in <module>
    main()
  File "/Users/apple/main.py", line 5, in main
    from request_llm.bridge_all import predict
ModuleNotFoundError: No module named 'request_llm'

**Comment by Sqyht1** [comment]
已解决

**Reply by dynamor2019** [reply]
如何解决的，我总是要重新安装一次才能正常运行
ModuleNotFoundError: No module named 'distutils'
